README.md

```diff
@@ -89,6 +89,34 @@ ReactDom.render(
 );
 ```
 
+## Troubleshooting
```

Comment:
Remove all this.
As you can see on https://demo.api-platform.com generator works without any problem.
It is the same case for any other properly working API-Platform based API.